usdf/storage.rst

```diff
@@ -8,40 +8,49 @@ Storage Locations
  
 Web space:
- - "public_html" web access is available upon request: it will be visible at https://s3df.slac.stanford.edu/people/<user>
- - send email to usdf-help at slac.stanford.edu requesting a personal web directory
- - a public_html directory will be created in your home directory after acknowledging a usage policy
- - note that this is for static pages - no server-side content.
- - no symlinks outside public_html will be followed.
-
-Science data under /sdf/group/rubin/:
- - datasets/
- - lsstdata/offline/ (still in prep)
- - repo/
- - shared/data/test_data - daily updated checkouts of git lfs repos for use in testing code (includes the ap_verify CI datasets)
- - ncsa_home - copy of NCSA home directories (in prep, setting permissions)
- - ncsa_jhome - copy of NCSA RSP home directories (in prep, setting permissions)
+ - "public_html" web access is available upon request: it will be visible at ``https://s3df.slac.stanford.edu/people/<user>``
+ - send email to ``usdf-help`` at ``slac.stanford.edu`` requesting a personal web directory
+ - a ``public_html`` directory will be created in your home directory after acknowledging a usage policy
+ - note that this is for static pages - no server-side scripting.
+ - no symlinks outside ``public_html`` will be followed.
+
+Science data under ``/sdf/group/rubin/``:
+ - Much of this is still disorganized and contains vestiges of the structure from NCSA that was moved to USDF.  Bear with us until we get it all straightened out.
+ - ``shared/`` contains various datasets for common usage.
+   - ``shared/data-repos.yaml`` contains the list of aliases for Butler repos.
+   - shared/data contains daily-updated test and validation datasets.
+   - Other requested storage for particular groups or external datasets is also located here.
+   - Request new shared storage via ``#ops-usdf``, providing the name, purpose, storage quota needed, and owner.  Storage will generally be given group ``rubin_users``, giving all staff access; this can be read-only or read-write as desired.
+ - ``repo/`` contains Butler repo configurations.  See the next section.
+ - ``sw/`` contains the "shared stack" of LSST Science Pipelines software.
  
- Shared stack builds
-  - /sdf/group/rubin/sw/
-  
-Interim while NCSA data is being placed at SLAC
- - SDF Lustre is separate from s3df filesystems. Access the Lustre filesystem via ``/fs/ddn/sdf/`` to access your SDF home directories, sandbox and scratch files
- - datasets and repo/main are read-only, with the exception of repo/main_20220411, which is live and in use for HSC reprocessing
-
 Butler access
 =============
 
-The primary Butler repos are located at ``/sdf/group/rubin/repo/main_20210215`` and ``/sdf/group/rubin/repo/dc2_20210215``.
-They can also be accessed via aliases ``/repo/main`` and ``/repo/dc2``.
+Butler repo configurations are located under ``/sdf/group/rubin/repo/``, but we define aliases for them starting with ``/repo/`` as a convenience.
```

Comment:
Nitpicky but the first part is not exactly true. Some of the (source-of-truth) repo configurations are at their S3 bucket locations, not under ``/sdf/group/rubin/repo/``